packaging/flavorgen/flavors/crs/csi.go

```diff
@@ -36,76 +31,88 @@ import (
 // CreateCrsResourceObjectsCSI creates the api objects necessary for CSI to function.
 // Also appends the resources to the CRS.
 func CreateCrsResourceObjectsCSI(crs *addonsv1.ClusterResourceSet) []runtime.Object {
-	serviceAccount := cloudprovider.CSIControllerServiceAccount()
-	serviceAccount.TypeMeta = metav1.TypeMeta{
-		Kind:       "ServiceAccount",
-		APIVersion: corev1.SchemeGroupVersion.String(),
-	}
-	serviceAccountSecret := newSecret(serviceAccount.Name, serviceAccount)
-	appendSecretToCrsResource(crs, serviceAccountSecret)
+	csiNamespace := cloudprovider.CSIDriverNamespace()
```

Comment:
not: why not group all csi relevant objects to a single configmap?

Reply:
All the content merged into a new `csi-manifests` config map now.